docs/Air.md

```diff
@@ -434,3 +435,19 @@ Gets pnr information and tickets list from [`importPNR`](#importPNR) and then do
 | ignoreTickets | `Boolean` | Defines if tickets should be ignored. The default value is `false` |
 
 **See: <a href="../examples/Air/cancelBooking.js">cancelBooking example</a>**
+
+## .addSegments(params)
+<a name="addSegments"></a>
+
+Add segments to an existing reservation record.
+
+**Returns**: `Promise` which is resolved with response message.
+
+| Param | Type | Description |
+| --- | --- | --- |
+| pnr | `String` | PNR |
+| version | `Number` | Current uAPI record version |
+| universalRecordLocatorCode | `String` | uAPI universal locator code |
```

Comment:
Are `universalRecordLocatorCode` and `reservationLocatorCode` retrievable by PNR?
If yes I would like to make them optional. If they are not provided, would be fin to extract them on our side.